src/infinite-loading/useInfiniteStoriesLoading.ts

```diff
@@ -10,9 +10,11 @@ import { useInfiniteLoading } from './useInfiniteLoading';
 async function fetchStories<T extends Story = Story>(
     page: number,
     pageSize: number,
+    useHighlightedStory: boolean,
     category?: Category,
     locale?: LocaleObject,
     include?: (keyof Story.ExtraFields)[],
+    pinning?: boolean,
```

Comment:
@e1himself I didn't notice that when reviewing your PR, but `pinning` was not taken into account for subsequent infinite loading of stories. I fixed it in this PR, as it was related.